doc/building.md

```diff
@@ -24,7 +24,8 @@ running on Windows), and want to clone the main-line JDK repository.
     `make images`
 
  4. Verify your newly built JDK: \
-    `./build/*/images/jdk/bin/java -version`
+    `./build/*/images/jdk/bin/java -version` \
+    Note the built JDK is at `/images/jdk` instead of `/jdk`.
```

Comment:
I'm not sure this is really helpful, or at least not in this place and form. Can we move the changes related to how to distinguish a full JDK image from a "partial JDK"/"exploded image" to a separate issue?